lighthouse-core/test/results/sample_v2.json

```diff
@@ -3537,51 +3523,51 @@
               "path": "3,HTML,1,BODY,36,A",
               "selector": "body > a",
               "nodeLabel": "external link",
-              "snippet": "<a href=\"https://www.google.com/\" target=\"_blank\">"
```

Comment:
Looks like some snippets are not being returned as well

Reply:
planning on merging this after #11405! these sample_v2 errors are coming from there, should be fixed over there soon